pkg/components/metallb/manifests.go

```diff
@@ -440,9 +440,182 @@ data:
       "editable": true,
       "gnetId": null,
       "graphTooltip": 0,
-      "id": 26,
       "links": [],
       "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "Prometheus",
+          "description": "Rate of Kubernetes object updates that failed for some reason.",
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 10,
+            "x": 0,
+            "y": 0
+          },
+          "hiddenSeries": false,
+          "id": 6,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "dataLinks": []
+          },
+          "percentage": false,
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "rate(metallb_k8s_client_update_errors_total[1m])",
```

Comment:
Maybe we could modify the label for those metrics, to only include pod name?

Reply:
Do you mean like this: https://snapshot.raintank.io/dashboard/snapshot/ojNAWQiZmHARtXOTZgEBlzGtAbjF07go ?

Show only pod names in the legend?

Reply:
Yeah, I think it's more readable then, but I don't know the conventions, so if you think this is worse, please ignore :) Thanks!

Reply:
I think this is readable. Than before because earlier there was lot of data.